docs/sources/installation/upgrading.md

```diff
@@ -243,8 +243,12 @@ change the `[security]` setting `cookie_secure` to `true` and use HTTPS when `co
 
 ### PhantomJS removed
 
-PhantomJS was deprecated in [Grafana v6.4](https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/#phantomjs-deprecation) and starting from Grafana v7.0.0, all PhantomJS support has been removed. This means that Grafana no longer ships with a built-in image renderer, and we adwise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
+PhantomJS was deprecated in [Grafana v6.4]({{< relref "../guides/whats-new-in-v6-4.md#phantomjs-deprecation" >}}) and starting from Grafana v7.0.0, all PhantomJS support has been removed. This means that Grafana no longer ships with a built-in image renderer, and we adwise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
 
 ### Dashboard minimum refresh interval enforced
 
 A global minimum dashboard refresh interval is now enforced and defaults to 5 seconds. Read more [here]({{< relref "configuration/#min-refresh-interval" >}}) about this setting.
+
+### Backend plugins
+
+Grafana now require backend plugins to be signed. If a backend plugin is not signed Grafana will not load/start it. This is an additional security measure to make sure backend plugin binaries and files haven't been tampered with. It's possible to allow unsigned plugins using a configuration setting, but is something we strongly disadvise doing. Read more [here]({{< relref "configuration/#allow-loading-unsigned-plugins" >}}) about this setting.
```

Comment:
How do I know if the backend plugins I am current using are signed?

Reply:
Don't see it on https://grafana.com/grafana/plugins?orderBy=weight&direction=asc , I think I saw it in Grafana, but by then I think you would have had to already have upgraded.

Reply:
You'll notice when Grafana starts. It logs an error `plugin <plugin id> is unsigned`. Enough if I add a note about that? 

I can also add "All backend plugins authored by Grafana Labs is signed"?

Reply:
"All backend plugins authored by Grafana Labs is signed" sounds good. I believe we are looking to add signing of community plugins at some point - 7.1? Should we mention that as well - seems like a natural follow up question.

Reply:
The logging seems more like a docs detail thing to me. The reason I prefer the other comment is that helps a user before they have upgraded, where as the log message only helps after they have upgraded.